lib/razor/cli/command.rb

```diff
@@ -41,8 +41,10 @@ def extract_command
     body = {}
     until @segments.empty?
       argument = @segments.shift
-      if argument =~ /\A--([a-z-]+)(=(.+))?\Z/
-        # `--arg=value` or `--arg value`
+      if argument =~ /\A--([a-z-]{2,})(=(.+))?\Z/ or
+          argument =~ /\A-([a-z])(=(.+))?\Z/
+        # `--arg=value`/`--arg value`
+        # `-a=value`/`-a value`
```

Comment:
It's surprising to me that razor does its own argument parsing. Is there no argument parsing library for rubby that does what you need?

About the functionality here, this does preclude the common pattern of being able to specify multiple short flags at once, like `netstat -apln` instead of having to type `netstat -a -p -l -n`, which I would argue is "proper short form argument style". Is it prohibitive to support this?